kafka/src/test/java/org/axonframework/extensions/kafka/eventhandling/consumer/AsyncFetcherTest.java

```diff
@@ -63,20 +59,58 @@
  * @author Nakul Mishra
  * @author Steven van Beelen
  */
-@DirtiesContext
-@ExtendWith(SpringExtension.class)
-@EmbeddedKafka(topics = {"testStartFetcherWith_ExistingToken_ShouldStartAtSpecificPositions"}, partitions = 5)
-class AsyncFetcherTest {
+
+class AsyncFetcherTest extends KafkaContainerTest {
 
     private static final String TEST_TOPIC = "some-topic";
     private static final int TEST_PARTITION = 0;
 
-    @SuppressWarnings("SpringJavaAutowiredMembersInspection")
-    @Autowired
-    private EmbeddedKafkaBroker kafkaBroker;
-
     private AsyncFetcher<String, String, KafkaEventMessage> testSubject;
 
+    @BeforeAll
+    public static void before() {
+        KafkaAdminUtils.createTopics(KAFKA_CONTAINER.getBootstrapServers(), TEST_TOPIC);
```

Comment:
Nit: `KAFKA_CONTAINER.getBootstrapServers()` can be moved to `KafkaContainerTest`as a base method that the tests can use without caring about the Container at all